text/0000-2020-roadmap.md

```diff
@@ -0,0 +1,120 @@
+- Title: `2020-roadmap`
+- Authors: [@lehnberg](mailto:daniel.lehnberg@protonmail.com)
+- Start date: `Jan 10, 2020`
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000)
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+
+[summary]: #summary
+
+This RFC lists areas and objectives that members of the Grin community would like to prioritize in 2020. The theme for the year is _**Underpinning**_. In construction, underpinning is the process of strengthening the foundation of an existing building or other structure. The ambition is to solidify Grin's foundation throughout the course of 2020 and make it easier to build on in future years. This list is **not exhaustive**, and reflecting Grin's open source nature, contributions in other areas are welcomed.
+
+# Motivation
+
+[motivation]: #motivation
+
+Having a list of priorities commonly agreed on by members of the community can facilitate planning and identify what some of the deliverables might be throughout the course of the year. It also serves as a way to measure progress.
+
+Furthermore, the process can be helpful in and of itself. As participants attempt to agree on a list of items that they consider are important to target for next year, it also becomes clear what is _not as important_ in this context.
+
+# Community-level explanation
+
+[community-level-explanation]: #community-level-explanation
+
+## Development
+
+- **Transaction building.** Establish an approach that will be robust and good enough be a viable baseline for building grin transactions, and can replace unsafe http methods.
```

Comment:
- **Kernal types.** Complete specification of and implement relative kernels as outlined in https://github.com/mimblewimble/grin-rfcs/pull/19